tools/fs/files.ts

```diff
@@ -193,6 +193,8 @@ export function usesWarehouse() {
   // Test hook: act like we're "installed" using a non-homedir warehouse
   // directory.
   if (process.env.METEOR_WAREHOUSE_DIR) {
```

Comment:
Both paths now return `!inCheckout()`, so I guess we should simplify the whole function to just that. Basically, "it uses warehouse when not in checkout". Doc comment should be update accordingly. Also commit message talks about `METEOR_WAREHOUSE_METADATA_DIR` which does not exist yet, so we could use a better message.

Reply:
Fixed